### DISCUSSION TITLE
[v3] Wizard last step tab check

### DISCUSSION THREAD

**Discussion by arnaudome** [filamentphp/filament#7955] [General]
Using the Wizard, is it possible to make the **last tab icon a check** (like the previous tabs) **when submitting the form** in order to show a View like Thanks for answering the form...etc...

**Ideas that come to my mind:**
- Adding a View step and somehow hide the tab. 
- Changing the icon of the last step tab depending if the form is submitted or not. (Tried this one but the check icon I put has the background transparent while the completed steps' icons have primary color background)

**Comment by arnaudome** [comment]
It seems the only option is to do some CSS.

**Reply by danharrin** [reply]
Once the form is submitted, redirect to a different page with feedback?

Otherwise, the user might exit the page and not actually submit the last step

**Reply by arnaudome** [reply]
My form is isolated and that's the reason why I don't want to redirect to any page.

The user will answer the last step because there is the submit button and the icon isn't checked until it is submitted.

I already did it and works fine.
